### ISSUE TITLE
Age bias correction 

### ISSUE THREAD

**Issue by anbai106** [hamiltonoh/OrganAge#9]
Hi,

In your Nature paper, were all the metrics you reported (MAE and Pearson's r) age-bias-corrected or not? 

Age bias correction: https://pmc.ncbi.nlm.nih.gov/articles/PMC7049655/

Thanks!

**Comment by hamiltonoh** [comment]
Hi, we did not do any corrections before calculating r/MAE. 

Regarding the age gap, we calculated it as the residual between predicted age and chronological age, instead of using the raw predicted age minus chronological age.

**Comment by anbai106** [comment]
Thanks for the explanation! Did you observe any overfitting of the model if you did not add the regularization term to your Lasso? Say if you train the age prediction on >2000 proteins.

**Comment by hamiltonoh** [comment]
Yes, generally we see more overfitting without regularization, especially when training sample size is low.

**Comment by anbai106** [comment]
Do you have any figures/results included in the Nature manuscript where the overfitting happens? i.e., the magnitude in relate to the sample size and number of proteins? I am running some analyses and found that overfitting phenomenon is related to both sample size and also number of proteins included.

**Comment by hamiltonoh** [comment]
Sorry no we don't have that in the manuscript. Your results make sense though.

**Comment by anbai106** [comment]
Thank you very much for your explanations! For age bias correction, I think as long as the downstream models include age as a covariate, it will be OK! In the neuroimaging community, as stated in the Commentary, age bias correction is always applied before downstream analyses! 

You can close this!

**Comment by anbai106** [comment]
Hi, I just thought about the age bias correction vs. residual-based age gap again. Statistically, I think they are very similar, which does both to alleviate the bias of "regressor towards the mean". Check this Nature Review Genetics paper: https://www.nature.com/articles/s41576-024-00807-w vs. this age bias correction: https://pmc.ncbi.nlm.nih.gov/articles/PMC7049655/

**Comment by anbai106** [comment]
Hi, do you mind pointing me to where you implemented the residual-based approach using the lowess function?

I think I found it

Thanks
